h/app.py

```diff
@@ -94,6 +121,13 @@ def includeme(config):
         },
     })
 
+    # After commit events
+    config.add_request_method(EventQueue, name='after_commit_events', reify=True)
```

Comment:
Not fully up-to-speed with the context here, but I'd suggest keeping "private" (i.e. implementation-specific) data structures on the registry, and "public" interfaces (such as for enqueueing an event) on the request object as request methods.